presto-main/src/main/java/com/facebook/presto/sql/gen/CursorProcessorCompiler.java

```diff
@@ -46,6 +49,9 @@
 import static com.facebook.presto.bytecode.Parameter.arg;
 import static com.facebook.presto.bytecode.ParameterizedType.type;
 import static com.facebook.presto.sql.gen.BytecodeUtils.generateWrite;
+import static com.facebook.presto.sql.gen.TryCodeGenerator.defineTryMethod;
+import static com.google.common.collect.Iterables.getOnlyElement;
+import static com.google.common.primitives.Primitives.wrap;
```

Comment:
I searched the code base. There is one case where we statically imported this. But we don't in other places. I personally don't like statically importing `Primitives.wrap` because `wrap` by itself doesn't convey what it's doing.